docs/man_pages/lib-management/kendoui-install.md

```diff
@@ -0,0 +1,51 @@
+kendoui install
+==========
+
+Usage | Synopsis
+------|-------
+List all versions | `$ appbuilder kendoui install [--verified] [--latest]`
```

Comment:
My concern is more for the cases where someone needs to revert to a version that is just not available via these flags and can only be reached through the interactive console. What happens then? (Not that it's very common or that anyone has reported it :D)